### PR TITLE
Minor changes after audit.

### PR DESCRIPTION
**Pull request by kantp**

### CODE REVIEW

File: shelley/design-spec/delegation_design_spec.tex [hunk 1]
```diff
@@ -1655,7 +1663,7 @@ \subsection{Block Validity and Operational Key Certificates}
 
 \begin{itemize}
 \item there is a stake pool with (cold) pool key \(vk_\text{leader}\), which
-  list \hash{sk_\text{VRF}} as the VRF key hash in its pool registration
+  list \hash{vk_\text{VRF}} as the VRF key hash in its pool registration
```
Comment (kantp) [inline shelley/design-spec/delegation_design_spec.tex:1666]: @JaredCorduan Could you double-check that we had this correct in the executable spec (i.e., that we use the hash of the verification VRF key in the pool registration). I am actually very sure we do, but still :)

Comment (JaredCorduan) [reply shelley/design-spec/delegation_design_spec.tex:1666]: absolutely, always happy to check!

here is the `PoolParams` type, which is the data inside the pool registration cert, and you can see the type `Hash crypto (VerKeyVRF crypto)` https://github.com/input-output-hk/cardano-ledger-specs/blob/master/shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/TxData.hs#L305

**Review by JaredCorduan — APPROVED** [review]
LGTM!

File: shelley/design-spec/delegation_design_spec.tex (hunk 1 above)
Comment (kantp) [reply shelley/design-spec/delegation_design_spec.tex:1666]: Thanks, Jared!

**Pull request merged**